src/components/vm/nics/nicBody.jsx

```diff
@@ -215,26 +215,17 @@ export const NetworkTypeAndSourceRow = ({ idPrefix, onValueChanged, dialogValues
                                    </Content>}
                            />
                        }>
-                    <Radio id={`${idPrefix}-source-mode-vepa`}
-                        name="mode-vepa"
-                        isChecked={dialogValues.networkSourceMode == "vepa"}
-                        label="VEPA"
-                        onChange={() => onValueChanged('networkSourceMode', "vepa")} />
-                    <Radio id={`${idPrefix}-source-mode-bridge`}
-                        name="mode-bridge"
-                        isChecked={dialogValues.networkSourceMode == "bridge"}
-                        label={_("Bridge")}
-                        onChange={() => onValueChanged('networkSourceMode', "bridge")} />
-                    <Radio id={`${idPrefix}-source-mode-private`}
-                        name="mode-private"
-                        isChecked={dialogValues.networkSourceMode == "private"}
-                        label={_("Private")}
-                        onChange={() => onValueChanged('networkSourceMode', "private")} />
-                    <Radio id={`${idPrefix}-source-mode-passthrough`}
-                        name="mode-passthrough"
-                        isChecked={dialogValues.networkSourceMode == "passthrough"}
-                        label={_("Passthrough")}
-                        onChange={() => onValueChanged('networkSourceMode', "passthrough")} />
+                    {["vepa", "bridge", "private", "passthrough"].map(mode =>
+                        <Radio
+                            key={mode}
+                            id={`${idPrefix}-source-mode-${mode}`}
+                            name={`mode-${mode}`}
+                            isChecked={dialogValues.networkSourceMode == mode}
+                            // The label is not translated since the
+                            // documentation we link to is always in
+                            // English.
+                            label={<pre>{mode}</pre>}
+                            onChange={() => onValueChanged('networkSourceMode', mode)} />)}
```

Comment:
Do we not want to keep the capitalization at least? So something like this?
```suggestion
                    {["VEPA", "Bridge", "Private", "Passthrough"].map(mode => {
                        const lowercaseMode = mode.toLowerCase();
                        return <Radio
                            key={lowercaseMode}
                            id={`${idPrefix}-source-mode-${lowercaseMode}`}
                            name={`mode-${lowercaseMode}`}
                            isChecked={dialogValues.networkSourceMode == lowercaseMode}
                            // The label is not translated since the
                            // documentation we link to is always in
                            // English.
                            label={<pre>{mode}</pre>}
                            onChange={() => onValueChanged('networkSourceMode', lowercaseMode)} />
                        })}
```

Reply:
I suppose that doesn't work, as the `name` needs to be the actual lower-case API value. TBH I think the current approach is fine, "VEPA" isn't translated and "private" and "passthrough" could very well be translated.